x-pack/dockerlogbeat/docs/limitations.asciidoc

```diff
@@ -0,0 +1,11 @@
+[[log-driver-limitations]]
+[role="xpack"]
+== Known problems and limitations
+
+experimental[]
+
+This release of the {log-driver} has the following problems and limitations:
```

Comment:
A few things:

- Does not support (beta) spool to disk
- Complex config options can't be (easily) represented via `--log-opts`